### PR TITLE
toc: move glossary, and reorganize "reference" section

### PR DESCRIPTION
**Pull request by thaJeztah**
With these changes, the reference section navigation looks like this:

Screenshot 2020-03-13 at 16 54 43

Moving the "file formats" lower is just a suggestion, not sure yet if it's more logical, so open to feedback, and happy to change the position back 👍

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya PTAL

File: _data/toc.yaml [hunk 1]
```diff
@@ -340,7 +340,7 @@ reference:
       title: About versions and upgrading
     - path: /compose/faq/
       title: Frequently asked questions
-- sectiontitle: Command-Line Interfaces (CLIs)
+- sectiontitle: Command-Line reference
```
Comment (usha-mandya) [inline _data/toc.yaml]: Suggest updating it to Command-line reference

**Comment by usha-mandya** [comment]
This looks so neat now. Thanks for fixing this @thaJeztah.

On a related note - searching for `Docker Compose command line reference` brings up https://docs.docker.com/compose/reference/ as the first result. However, this is not linked to our TOC. Wondering if this is intentionally removed from the toc? We can skip this page from Google indexing if that's the case. Wanted to know your thoughts.

**Comment by thaJeztah** [comment]
@usha-mandya changed `Command-Line reference` to `Command-line reference` (lowercase `line`); PTAL 🤗

**Pull request merged**